src/styles/blocks/card.scss

```diff
@@ -0,0 +1,168 @@
+
+  padding: 40px 50px;
+
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 48px 46px;
+  justify-content: center;
+  align-items: center;
+}
+
+@media (min-width: 488px) {
+  .cards-container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .cards-container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards-container {
+    --column-count: 4;
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid $light-gray;
+  border-radius: 5px;
+  margin: 16px;
+  height: auto;
+  transition:
+    transform 300ms,
+    color 300ms;
+
+  &__image {
+    display: flex;
+
+    .image {
+      width: 160px;
+      height: 134px;
+      margin: 32px 19px 0;
+
+      background-image: url(../images/imac.jpeg);
+      background-size: cover;
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+  }
+
+  &__title {
+    margin: 40px 16px 0;
+    line-height: 18px;
+    height: 36px;
+    width: 166px;
+    font-weight: 500;
+    font-size: $font-size-medium;
+    color: $secondary-color;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    padding: 0 16px;
+    width: 100%;
+    justify-content: center;
+    font-weight: 400;
+    font-size: $font-size-small;
+    line-height: 14px;
+    color: $gray;
+  }
+
+  &__reviews {
+    display: flex;
+    margin: 16px;
+    margin-bottom: 0;
+    width: 166px;
+    justify-content: space-between;
+
+    .reviews {
+      font-weight: 400;
+      font-size: $font-size-small;
+      line-height: 16px;
+      color: $secondary-color;
+      text-align: right;
+    }
+  }
+
+  &__price {
+    display: flex;
+    width: 166px;
+    line-height: 18px;
+    justify-content: space-between;
+    margin: 24px 16px 0;
+
+    .price {
+      font-size: $font-size-medium;
+      font-weight: 400;
+      color: $gray;
+    }
+
+    .amount {
+      font-size: $font-size-extra-large;
+      font-weight: 700;
+      color: $secondary-color;
+    }
+  }
+
+  .button {
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+    display: flex;
+    width: 166px;
+    height: 40px;
+    margin: 16px;
+    border-color: $primary-color;
+    border-radius: 5px;
+    background: $primary-color;
+    transition:
+      background-color 300ms,
+      border 300ms;
+
+    &__buy {
+      box-sizing: border-box;
+      width: 100%;
+      height: 100%;
+      justify-content: center;
+      align-items: center;
+      display: flex;
+
+      text-decoration: none;
+      text-transform: uppercase;
+      color: white;
+      font-size: $font-size-large;
+      font-weight: 700;
+      transition: color 300ms;
+
+      &:hover {
+        color: $primary-color;
+      }
+    }
+
+    &:hover {
+      background-color: white;
+      border: 1px solid $primary-color;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The scale transformation on hover might cause layout shifts. Ensure that the surrounding layout can accommodate this change without causing overflow or unexpected shifts.